tests/js/test.js

```diff
@@ -0,0 +1,70 @@
+/*
+
+A test script to look at how Nightmare behaves natively.  This should not be used.
+
+ */
+"use strict";
+var Nightmare = require("nightmare");
+
+var path = require("path");
+var url = require("url");
+
+var startPage = url.resolve(url.resolve("file://", __dirname), "./static/html/select.html");
+
+//var browser = Nightmare({ show: true});
```

Comment:
I removed the test file, I mostly needed it early on when wrestling with the Windows implementation.